add_links.sh

```diff
@@ -1,9 +1,25 @@
 #!/usr/bin/env bash
 
-mkdir docs/2_20_tmp
-(cd docs/2_20; tar cf - .) | (cd docs/2_20_tmp; tar xvf - )
-python link_to_latest.py docs/2_20_tmp/stan-users-guide "Stan User's Guide"
-python link_to_latest.py docs/2_20_tmp/reference-manual "Stan Reference Manual"
-python link_to_latest.py docs/2_20_tmp/functions-reference "Stan Functions Reference"
-mv docs/2_20 docs/2_20_bak
-mv docs/2_20_tmp/ docs/2_20
+### USAGE
+# add_links.sh 
+
+cd docs
+# Count number of dirs in /docs
+docsDirsCount=`find . -mindepth 1 -maxdepth 1 -type d | wc -l`
+# Get [-1] docs version
+# Substract functions-reference, reference-manual, stan-users-guide, current version and remove trailing /
```

Comment:
this is kind of clunky - why not just pass in $major and $minor version and calculate correct version in Jenkinsfile

Reply:
This looks simple to me, as in passing the version I would have to do minor -1 and consider cases where major = 3 and minor = 00, I would have no way to properly check what was the last minor. But give me a minute, I'll make it nicer.

Reply:
@mitzimorris I've refactored the above. Sorry for a long time but I just saw this in the backlog.